src/07_slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print((a[1]))
 
 # Output the second-to-last element: 9
-print()
+print(a[4])
 
 # Output the last three elements in the array: [7, 9, 6]
-print()
+print(a[3:5])
```

Comment:
Remember that the last index in a slice is not included in the result, so this will only print indices 3 and 4.  
to get the last x elements in a list, we can use `print(a[x:])`

Reply:
thanks @notontilt09 for the edit!